pkg/southbound/clientManager.go

```diff
@@ -43,24 +43,24 @@ func createDestination(device topocache.Device) (*client.Destination, DeviceID)
 	d.Target = device.Target
 	d.Timeout = time.Duration(device.Timeout) * time.Second
 	if device.Plain {
-		log.Println("Plain connection connection to", device.Addr)
+		log.Warning("Plain connection connection to", device.Addr)
```

Comment:
I woudl say info, these only stem from elements that the user gave, thus it's well known

same for line 48

Reply:
done